test/e2e_env/universal/envoyconfig/envoyconfig.go

```diff
@@ -61,17 +64,8 @@ func EnvoyConfigTest() {
 
 	E2EAfterEach(func() {
 		// delete all meshtimeout policies
-		out, err := universal.Cluster.GetKumactlOptions().RunKumactlAndGetOutput("get", "meshtimeouts", "--mesh", meshName, "-o", "json")
-		Expect(err).ToNot(HaveOccurred())
-		var output struct {
-			Items []struct {
-				Name string `json:"name"`
-			} `json:"items"`
-		}
-		Expect(json.Unmarshal([]byte(out), &output)).To(Succeed())
-		for _, item := range output.Items {
-			Expect(universal.Cluster.GetKumactlOptions().RunKumactl("delete", "meshtimeout", item.Name, "--mesh", meshName)).To(Succeed())
-		}
+		Expect(cleanAllPoliciesOfType(meshtimeout.MeshTimeoutResourceTypeDescriptor, meshName)).To(Succeed())
```

Comment:
Can't we use `DeleteMeshResources` ?https://github.com/kumahq/kuma/blob/master/test/framework/resources.go#L20

Reply:
we can, I forgot about this function